frontend/amundsen_application/static/js/pages/SearchPage/SearchFilter/ToggleFilter/index.tsx

```diff
@@ -0,0 +1,83 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import { GlobalState } from 'ducks/rootReducer';
+import {
+  updateFilterByCategory,
+  UpdateFilterRequest,
+} from 'ducks/search/filters/reducer';
+
+import InfoButton from 'components/InfoButton';
+
+import ToggleSwitch from 'components/ToggleSwitch/ToggleSwitch';
+
+import './styles.scss';
+
+interface OwnProps {
+  categoryId: string;
+  filterName: string;
+  helpText?: string;
+}
+
+interface StateFromProps {
+  checked: boolean;
+}
+
+interface DispatchFromProps {
+  applyFilters: (categoryId: string, value: string[]) => UpdateFilterRequest;
+}
+// TODO change to FC
+
+export type ToggleFilterProps = OwnProps & DispatchFromProps & StateFromProps;
+
+export class ToggleFilter extends React.Component<ToggleFilterProps> {
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
I don't think you need this as you are using an arrow function in the 'handleChange' method.